bionemo-recipes/models/geneformer/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM nvcr.io/nvidia/pytorch:25.06-py3
```

Comment:
can we make this at least 25.09? preferably 25.10

Reply:
Fixed